source/accessible-media/formats.html.erb.md

```diff
@@ -15,79 +14,38 @@ contributors:
--   [Web Video Text Tracks (WebVTT)](https://w3c.github.io/webvtt/)
--   [SRT](https://matroska.org/technical/specs/subtitles/srt.html)
+-   [Timed Text Markup Language (TTML)](https://www.w3.org/TR/ttaf1-dfxp/) <br>
+    Support only in Internet Explorer 10+ and Edge, as well as many stand-alone players. Streaming media services typically use TTML to convey captions to users.
 
-Captions can be displayed by most modern media players on desktop and mobile platforms, in standalone applications as well as in Web browsers:
+-   [SRT](https://matroska.org/technical/specs/subtitles/srt.html) <br>
+    SRT is not supported natively by any browser, but is supported by most
+    other types of media players including those provided by popular
+    video-hosting services, some social-media platforms and by custom
+    players.
 
--   Standalone players typically support WebVTT and/or TTML.
--   Streaming media services typically use TTML to convey captions to users.
--   Web browsers support various caption formats, as shown in the table below.
-
-
-<table>
-	<tr>
-		<th scope="col">Browser</th>
-		<th scope="col">OS</th>
-		<th scope="col">Supported caption format(s)</th>
-	</tr>
-	<tr>
-		<th scope="row">Firefox</th>
-		<td>Windows, OS X, Android, iOS</td>
-		<td>WebVTT</td>
-	</tr>
-	<tr>
-		<th scope="row">IE 10, 11; Edge</th>
-		<td>Windows</td>
-		<td>TTML, WebVTT</td>
-	</tr>
-	<tr>
-		<th scope="row">Safari</th>
-		<td>OS X; iOS</td>
-		<td>WebVTT</td>
-	</tr>
-	<tr>
-		<th scope="row">Chrome</th>
-		<td>Windows, OS X, Chrome OS, Android, iOS</td>
-		<td>WebVTT</td>
-	</tr>
-</table>
-
-SRT is not supported natively by any browser, but is supported by most
-other types of media players including those provided by popular
-video-hosting services, some social-media platforms and by custom
-players.
-
-WebVTT, TTML and SRT are "sidecar" files, which is to say they are
-transmitted separately from their corresponding video files (riding
-alongside the video data in the delivery stream, rather than being
-embedded directly into the video file), and are synchronized and
+WebVTT, TTML and SRT captions are files that are stored and loaded
+separately from their corresponding video files. They are synchronized and
 displayed by the user agent at the time of playback.
 
-## Delivering captions to viewers
+## Using Captions
 
-Captions are delivered to viewers using HTML5's `track` element, which
-was created specifically for carrying text tracks, such as captions,
-subtitles and [text-based audio descriptions](g). `track` is used as a
-child element of the `video` element:
+Captions are associated with videos using HTML5’s `<track>` element. It is used as a
+child of the `<video>` element:
 
 {::nomarkdown}
 <%= code_start %>
 {:/nomarkdown}
 
 ~~~html
 <video controls>
-    <source src="myvideo.mp4" type="video/mp4" />
-        <track kind="captions" src="myvideo_captions.vtt" srclang="en" label="Captions" default />
```

Comment:
added information about the kind and label attributes; included a link to the attribute list in the HTML5.1 rec.